pkgs/servers/dns/knot-resolver/default.nix

```
@@ -26,6 +27,16 @@ unwrapped = stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
 
+  patches = [
+    (fetchpatch {
+      name = "fix-config-tests-on-darwin.patch";
+      url = "https://gitlab.nic.cz/knot/knot-resolver/-/commit/48ad9d436cf80f58c107774c313a561d852148a0.diff";
+      sha256 = "CEX1XkeYLUSe31xUhNdMRMl1VUXtKFCs5noNJaqL5x0=";
+    })
+  ] ++ lib.optionals (stdenv.isDarwin && stdenv.isAarch64) [
+    ./tests1.patch
+  ];
+
   # Path fixups for the NixOS service.
   postPatch = ''
     patch meson.build <<EOF
```

```suggestion
  postPatch = ''
    # Path fixups for the NixOS service.
    patch meson.build <<EOF
```

Side note: I am not really happy that we inline patch files.

---

I don't feel strongly about that, for such trivial patches.  Fortunately there's no need to splice nix values in here, contrary to the `ffi.load` below.